node/grpc/server_v2.go

```diff
@@ -166,6 +184,15 @@ func (s *ServerV2) GetChunks(ctx context.Context, in *pb.GetChunksRequest) (*pb.
 		return nil, api.NewErrorInternal(fmt.Sprintf("failed to get chunks: %v", err))
 	}
 
+	size := 0
+	if len(chunks) > 0 {
+		size = len(chunks[0]) * len(chunks)
+	}
+	s.metrics.ReportGetChunksDataSize(size)
+
+	elapsed := time.Since(start)
+	s.metrics.ReportGetChunksLatency(elapsed)
```

Comment:
It seems "elapsed" can be inlined to Report call

Reply:
done